Jenkinsfile

```diff
@@ -22,7 +22,7 @@ pipeline {
                     // Show the agent name in the build list
                     echo env.NODE_NAME
                     env.MAVEN_HOME = tool('Maven')
-                    env.MAVEN_OPTS = "-Xmx1g -XX:+HeapDumpOnOutOfMemoryError"
+                    env.MAVEN_OPTS = "-Xmx1500m -XX:+HeapDumpOnOutOfMemoryError"
```

Comment:
We currently use 1200 on `main`. Should we increase there as well?

Reply:
Yes, I'll do that.
 I've added one more change: the missing `url` and `repositoryId` :crossed_fingers:

Reply:
created: https://github.com/infinispan/infinispan/pull/11204